Wabbajack/Installer.cs

```diff
@@ -634,7 +634,7 @@ private string HashArchive(string e)
             return HashArchive(e);
         }
 
-        public static string CheckForModList()
+        public static ModList CheckForModList()
```

Comment:
In the current HEAD 3e6dd46803c2ad7b040e7b32d6b297cf2d68c969 changed `CheckForModPack` from string to ModList, resolved the conflict for that reason